episodes/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  nixpkgs ? builtins.getFlake "nixpkgs",
+  system ? builtins.currentSystem,
+  pkgs ? nixpkgs.legacyPackages.${system},
+  lib ? nixpkgs.lib,
+}: let
+  episodes = lib.pipe ./. [
+    builtins.readDir
+    # filter directories
+    (elems: builtins.filter (name: elems.${name} == "directory") (builtins.attrNames elems))
```

Comment:
Let's have episode info in a top-level directory on their own, not right next to files from the tool, so that we don't need such a filter here